package.json

```diff
@@ -133,6 +132,7 @@
     "danger": "^13.0.4",
     "date-fns-jalali-v2": "npm:date-fns-jalali@2.30.0-0",
     "date-fns-v2": "npm:date-fns@2.30.0",
+    "esbuild": "^0.24.2",
```

Comment:
Could we use the latest version to avoid redundant dependency addition and add it to the `renovate` `Vite & Vitest` group?